src/applications/vaos/tests/fixtures/MockReferralDraftAppointmentResponse.js

```diff
@@ -227,13 +229,18 @@ class MockReferralDraftAppointmentResponse {
       return MockReferralDraftAppointmentResponse.create500Response();
     }
 
-    // Create slots array
+    // Create slots array with all dates in the next month
     const slotsArray = [];
-    const startHour = 16; // Starting at 4 PM UTC
+    const startHour = 14; // Starting at 2 PM UTC
+
+    // Get first day of next month
+    const today = new Date();
+    const firstDayNextMonth = addMonths(setDate(today, 1), 1);
 
     for (let i = 0; i < numberOfSlots; i++) {
-      const slotDate = addDays(new Date(), i + 3);
-      slotDate.setHours(startHour + i, 0, 0, 0);
+      // Create slots on consecutive days starting from the first day of next month
```

Comment:
Note: Now all appointment slots will be in one month, at the beginning of the month, so they will no traverse multiple months regardless of the day the test it run.